package/README.md

```diff
@@ -97,6 +97,8 @@ function makeClient() {
     // you can disable result caching here if you want to
     // (this does not work if you are rendering your page with `export const dynamic = "force-static"`)
     fetchOptions: { cache: "no-store" },
+    // you can override the default set here for individual queries in your components by calling them like
+    // const { data } = useSuspenseQuery(MY_QUERY, { context: { fetchOptions: { cache: "force-cache" }}});
```

Comment:
Some minor nits: calling out this goes in `context` so no one misses that, and should is `to an Apollo Client data fetching hook` or `to an Apollo Client suspense hook` more correct here? I went with the former because I thought this might still be relevant to `useQuery` calls.

```suggestion
    // you can override the default `fetchOptions` on a per query basis
    // via the `context` property on the options passed as a second argument
    // to an Apollo Client data fetching hook, e.g.:
    // const { data } = useSuspenseQuery(MY_QUERY, { context: { fetchOptions: { cache: "force-cache" }}});
```